input/reactive-extensions/Add/CompositeDisposable.Add.md

```diff
@@ -0,0 +1,67 @@
+# CompositeDisposable.Add Method
+
+Adds a disposable to the CompositeDisposable or disposes the disposable if the CompositeDisposable is disposed.
+
+**Namespace:**  [System.Reactive.Disposables](System.Reactive.Disposables\System.Reactive.Disposables.md)  
```

Comment:
Namespace links seem to be broken, if I compile the website and click them, I get 404. Not sure to which location should those links point a user. Maybe worth making them not clickable?

Reply:
as mentioned by @daveaglick 
Set DocsKeys.ValidateRelativeLinks and it should run a link validator on all pipelines
That'll give you warnings, also set DocsKeys.ValidateLinksAsError to make invalid links generate errors and fail the site build
Lets use that to validate our links in our pages for this.

Reply:
Also I think Microsoft was attempting to achieve something like this with those namespace links https://docs.microsoft.com/en-us/dotnet/api/microsoft.csharp.runtimebinder?view=netcore-2.2

Reply:
I can go through the pages again and try and make sure all the links resolve to docs.  Some of that is just how the pages were delivered from Microsoft.

@glennawatson You bring up a good point.  Should we be failing builds for dead hyper links?  I believe it will help us keep the overall health of the site better.

Reply:
@glennawatson interestingly enough 
when I add `Settings[DocsKeys.ValidateLinksAsError] = "true";` the build passes
when I add `Settings[DocsKeys.ValidateRelativeLinks] = "true";` the build fails.

Reply:
DocsKeys.ValidateAbsoluteLinks: System.Boolean

Set to true (the default value is false) to validate all absolute links. Note that this may add considerable time to your generation process.

DocsKeys.ValidateLinksAsError: System.Boolean

Set to true (the default value is false) to report errors on link validation failures.

DocsKeys.ValidateRelativeLinks: System.Boolean

Set to true (the default value) to validate all relative links.

Reply:
Turn those three to true I guess. If it doesn't slow down the build dramatically let's turn it on in a separate PR today